gsoc_summary.md

```diff
@@ -0,0 +1,82 @@
+The main issue for the project and the general discussion can be inspected here [issue 842](https://github.com/regro/cf-scripts/issues/842). The affected repositories can be found repectiviley here [cf-scripts](https://github.com/regro/cf-scripts), [cf-graph-countyfair](https://github.com/regro/cf-graph-countyfair) and [circle_worker](https://github.com/regro/circle_worker). 
+
+   - To begin with, I had to read the graph (it is load the graph to inspect the attribute of each feedstock), then get the new versions and of course bump it to somewhere else to not interfere with the graph structure [#1](https://github.com/viniciusdc/viniciusdc.github.io/issues/1);
+   - After this brief prelude of the code, I reached a more versatile code and approach that adapted the current code for the function that collects and requests the new versions (get_latest_version) and update the graph with those (update_upstream_versions), the main modification here also included a separation between the code that updates the versions (it is the actual computation) from the sources classes of the feedstocks. [#2](https://github.com/regro/cf-scripts/pull/1027);
+   - Now we have a new script called `populate_update_versions` that collects the new versions and works without disrupting the graph directly. We can start running the new process and check the information been sended to the `versions` folder at `regro/cf-graph-countyfair` [repository](https://github.com/regro/cf-graph-countyfair/), the process will then be running inside a continuous integration system (Circle CI) [#3](https://github.com/regro/circle_worker/pull/61);
+   - Due to the expensive time consuming usage of the actual and as every request of the sources new versions could be evaluated asynchronously inside the process I’ve added a pool feature to the update version code to increase the efficiency of the process [#4](https://github.com/regro/cf-scripts/pull/1049);
+   - Once the new versions folder was ready with the feedstocks versions data, the next thing to do was work at the code that gathered that information and inserted back to the graph, we've opted to create the feature, called `update_nodes_with_new_versions` inside the current `make_graoh` function as it main role was already build and upgrade the graph structure with data. [#5](https://github.com/regro/cf-scripts/pull/1050);
+   - As a matter of fact, we could also insert new utilities insethe the update_upstream_versions code to solve some issues [#6](https://github.com/regro/cf-scripts/pull/1073);
+   - Finally we could proceed and switch the old update upstream version code with the new one, and set the new process to work altogether with the main bot processes [#7](https://github.com/regro/cf-scripts/pull/1075), as also CJ helped me solving some bugs and other issues with the code [#8](https://github.com/regro/cf-scripts/pull/1099).
```

Comment:
CJ is famous in conda-forge but I'm not sure Google will know him without the full name ;-p